docs/data/joy/components/input/InputSubscription.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import FormControl from '@mui/joy/FormControl';
+import FormLabel from '@mui/joy/FormLabel';
+import FormHelperText from '@mui/joy/FormHelperText';
+import Input from '@mui/joy/Input';
+import Button from '@mui/joy/Button';
+
+export default function InputSubscription() {
+  const [form, setForm] = React.useState<{
+    email: string;
+    status: 'initial' | 'loading' | 'failure' | 'sent';
+  }>({
+    email: '',
+    status: 'initial',
+  });
+
+  const handleSubmit = () => {
+    setForm((current) => ({ ...current, status: 'loading' }));
+    try {
+      // Replace timeout with real backend operation
+      setTimeout(() => {
+        setForm({ email: '', status: 'sent' });
+      }, 1500);
+    } catch (error) {
+      setForm((current) => ({ ...current, status: 'failure' }));
+    }
+  };
+
+  return (
+    <FormControl>
+      <FormLabel
+        sx={(theme) => ({
+          '--FormLabel-color': theme.vars.palette.primary[400],
+        })}
+      >
+        MUI Newsletter
+      </FormLabel>
+      <Input
+        sx={{ pt: 0.5, pb: 0.5 }}
```

Comment:
```suggestion
```
padding should not be customized directly. I will think about how to prevent this.